Sources/TuistCore/Graph/Graph.swift

```diff
@@ -357,7 +357,7 @@ public class Graph: Encodable, Equatable {
             if let hostApp = hostApplication(for: targetNode) {
                 references.subtract(try embeddableFrameworks(path: hostApp.path, name: hostApp.name))
             } else {
-                references.subtract(precompiledFrameworks)
+                references = []
```

Comment:
Is there a test case for this? If so, could you modify it so it tests that locally built frameworks are not embedded, too?